simple_history/tests/tests/utils.py

```diff
@@ -26,13 +28,32 @@ def assertRecordValues(self, record, klass: Type[Model], values_dict: dict):
         :param klass: The type of the history-tracked class of ``record``.
         :param values_dict: Field names of ``record`` mapped to their expected values.
         """
-        for key, value in values_dict.items():
-            self.assertEqual(getattr(record, key), value)
-
-        self.assertEqual(record.history_object.__class__, klass)
-        for key, value in values_dict.items():
-            if key not in ("history_type", "history_change_reason"):
-                self.assertEqual(getattr(record.history_object, key), value)
+        values_dict_copy = values_dict.copy()
+        for field_name, expected_value in values_dict.items():
+            value = getattr(record, field_name)
+            if isinstance(value, Manager):
+                # Assuming that the value being a manager means that it's an M2M field
+                self._assert_m2m_record(record, field_name, expected_value)
+                # Remove the field, as `history_object` (used below) doesn't currently
+                # support historical M2M queryset values
+                values_dict_copy.pop(field_name)
+            else:
+                self.assertEqual(value, expected_value)
+
+        history_object = record.history_object
+        self.assertEqual(history_object.__class__, klass)
+        for field_name, expected_value in values_dict_copy.items():
+            if field_name not in ("history_type", "history_change_reason"):
+                self.assertEqual(getattr(history_object, field_name), expected_value)
+
+    def _assert_m2m_record(self, record, field_name: str, expected_value: List[Model]):
```

Comment:
might this be better named as `_assert_m2m_relation`? Or `_assert_m2m_queryset`? I don't mean to bikeshed, but as it currently stands, it makes it sounds like it's comparing a single value. However, it's really fetching all the related objects to compare. Having the name imply that it'll do a DB hit will be useful for the future.